llvm/include/llvm/ADT/SmallPtrSet.h

```diff
@@ -362,6 +364,22 @@ class SmallPtrSetImpl : public SmallPtrSetImplBase {
   template <typename UnaryPredicate>
   bool remove_if(UnaryPredicate P) {
```

Comment:
This should probably document what happens to iterators to explain what the difference is compared to (a sequence of) erase.